src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int BALL_AMOUNT = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[BALL_AMOUNT];
+        for (int ball = 0; ball < BALL_AMOUNT; ball++) {
```

Comment:
maybe `ball` isn't the best name for this variable
because actually it isn't a ball (like instance of class `Ball`), it is a number

Reply:
Agreet.
-Renamed in new commit.